cypress/e2e/login.cy.tsx

```diff
@@ -95,24 +93,23 @@ describe('Button Component', () => {
     cy.get('input[type="password"]').type(secretKey);
     cy.get('button[class="button button__complete"]').click();
     cy.wait(1000);
-    cy.contains('La llave secreta que a ingresado es incorrecta');
+    cy.contains('La llave secreta que ha ingresado es incorrecta');
   });
 
   it('Should show a message that the account has been activated when clicking on the activate account button', () => {
     cy.get('@checkboxInput').should('exist');
     cy.get('@checkboxInput').should('not.be.checked');
-    cy.contains('button', 'Continuar').should('exist');
+    cy.contains('button', 'Continuar').should('exist').as('buttonContinue');
```

Comment:
Remember what we talked about before, alias has to be declared in our `beforeEach`, that way we avoid declaring it in every test.